yaqd_wright/_wright_fuyu_linear.py

```diff
@@ -0,0 +1,72 @@
+__all__ = ["WrightFuyuLinear"]
+
+import asyncio
+from typing import Dict
+import time
+
+from yaqd_core import IsHomeable, HasLimits, IsDiscrete, HasPosition, UsesUart, UsesSerial, aserial
+
+
+class WrightFuyuLinear(IsHomeable, HasLimits, IsDiscrete, HasPosition, UsesUart, UsesSerial):
+    _kind = "wright-fuyu-linear"
+    serial_dispatchers: Dict[str, aserial.ASerial] = {}
+
+    def __init__(self, name, config, config_filepath):
+        super().__init__(name, config, config_filepath)
+        self._motornum = config["motor"]
+        self._units = config["units"]
+        self._microstep = config["microstep"]
+        self._steps_per_rotation = config["steps_per_mm"]
+        if config["serial_port"] in WrightFuyuLinear.serial_dispatchers:
+            self._serial_port = WrightFuyuLinear.serial_dispatchers[config["serial_port"]]
+        else:
+            self._serial_port = aserial.ASerial(config["serial_port"], config["baud_rate"])
+            WrightFuyuLinear.serial_dispatchers[config["serial_port"]] = self._serial_port
+        self._set_microstep(self._microstep)
+
+    def _set_position(self, position):
+        step_position = round(
```

Comment:
If I understand correctly, the -1 is just because your way of implementing the direction convention (e.g. if you flipped the stage 180, would you want to get rid of -1)?  If I've got it right, you probably want this detail in your config file (e.g. invert = True/False)

Reply:
i didn't see invert as a config so yes that would be good to do